content/experiments/portal-journey.md

```diff
@@ -1,7 +1,5 @@
 ---
-thumbnail: /portal-journey.png
-title: Portal Journey
-slug: portal-journey
+status: unpublished
```

Comment:
I agree with the solution, I think `draft` would be more accurate

Reply:
Ok. Changed `unpublished` to `draft`. 

Fwiw, if you set a `status` and it's anything but `published` it will result in the card not appearing on the landing page.